appsDiscovery/src/main/AndroidManifest.xml

```diff
@@ -7,5 +7,14 @@
         >
         <activity android:name="org.kinecosystem.appsdiscovery.sender.discovery.view.AppsDiscoveryActivity"
             android:screenOrientation="portrait"/>
+        <service
+            android:name="org.kinecosystem.appsdiscovery.receiver.service.ReceiveKinServiceBase"
+            android:exported="false">
+            <intent-filter>
+                <action android:name="KinReceiverTransactionCompleted"/>
```

Comment:
the convention for action name is FQN,  i.e. org.kinecosystem....KinReceiverTransactionCompleted

Reply:
cool